.github/pull_request_template.md

```diff
@@ -0,0 +1,52 @@
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+# Changes
```

Comment:
I'd recommend using Kubernetes' pull request template - https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md

The k8s template is used in a lot of other repos. Would be good to use it for consistency.